src/renderer/components/Text.tsx

```diff
@@ -0,0 +1,127 @@
+import { CSSProperties, PropsWithChildren } from 'react'
+import Typography from '@mui/material/Typography'
+
+type BaseProps = PropsWithChildren<{
+  align?: Extract<
+    CSSProperties['textAlign'],
+    'inherit' | 'left' | 'center' | 'right' | 'justify'
+  >
+  bold?: boolean
+  className?: string
+  id?: string
+  italic?: boolean
+  style?: CSSProperties
+  underlined?: boolean
+}>
+
+export function TitleText({
+  align,
+  bold,
+  children,
+  className,
+  italic,
+  style,
+  underlined,
+}: BaseProps) {
+  return (
+    <Typography
+      variant="h1"
+      align={align}
+      style={{
+        ...style,
+        fontWeight: bold ? 'bold' : undefined,
+        fontStyle: italic ? 'italic' : undefined,
+        textDecoration: underlined ? 'underline' : undefined,
+      }}
+      className={className}
+    >
+      {children}
+    </Typography>
+  )
+}
+
+export function BodyText({
+  align,
+  bold,
+  children,
+  className,
+  italic,
+  style,
+  underlined,
+}: BaseProps) {
+  return (
+    <Typography
+      variant="body1"
+      style={{
+        ...style,
+        fontWeight: bold ? 'bold' : undefined,
+        fontStyle: italic ? 'italic' : undefined,
+        textDecoration: underlined ? 'underline' : undefined,
+      }}
+      align={align}
+      className={className}
+    >
+      {children}
+    </Typography>
+  )
+}
+
+export function SubtitleText({
+  align,
+  bold,
+  children,
+  className,
+  italic,
+  style,
+  underlined,
+}: BaseProps) {
+  return (
+    <Typography
+      variant="subtitle1"
+      style={{
```

Comment:
applies to all variants, but seeing our custom `style` prop being mapped to MUI's `style` prop makes me wonder if we should update our `Button`  to do the same thing. Currently it exposes a `sx` prop that maps to MUI's `sx` prop, which has some MUI-specific magic going on (which we probably don't really need).

Mostly want to be consistent across our components when it comes to style overrides. I think just the exposing `style: CSSProperties` like this text component is doing feels safer to me. any thoughts?